### DISCUSSION TITLE
流加载：自定义‘加载更多’，建议把这个加进去，免得每次版本更新都要手动修改。

### DISCUSSION THREAD

**Discussion by mengxianseng** [layui/layui#1961] [建议]
// 流加载实例
					flow.load({
						elem: '#ID-PINGLUN-D751658755906748', // 流加载容器
						isAuto: true,
						next: '加载更多评论',//自定义‘加载更多’，建议把这个加进去，免得每次版本更新都要手动修改。
						end:' ',

//layui.js 修改此处
u = o.next === undefined ? "<cite>\u52a0\u8f7d\u66f4\u591a</cite>":o.next)+"</a></div>")//修改增加next属性 加载更多 的自定义样式

**Comment by sentsim** [comment]
欢迎提交 PR : https://github.com/layui/layui/pulls
